swap/htlc.go

```diff
@@ -122,8 +122,8 @@ var (
 	// QuoteHtlcP2TR is a template script just used for sweep fee
 	// estimation.
 	QuoteHtlcP2TR, _ = NewHtlcV3(
-		^int32(0), dummyPubKey, dummyPubKey, dummyPubKey, dummyPubKey,
-		quoteHash, &chaincfg.MainNetParams,
+		input.MuSig2Version100RC2, ^int32(0), dummyPubKey, dummyPubKey,
```

Comment:
This uses 1.0 while the other changes in the PR use 0.4. Is that on purpose?

Reply:
Yeah this PR is just making it possible to pass in the version if needed. We just need this specific dummy htlc for fee estimation so the version can be anything here.